src/util/helpers.tsx

```diff
@@ -290,3 +290,27 @@ export const getClientOS = (userAgent: string) => {
 
   return null;
 };
+
+export const getImageToTarballBody = (fileList: FileList): File | FormData => {
```

Comment:
This seems very specific to the ImportImageFromTarballForm, so I would keep it local to that file. This is very unlikely to be reused anywhere else.